internal/network/utils.go

```diff
@@ -239,3 +239,51 @@ func GetIPv4Networks(inventory *models.Inventory) (ret []string, err error) {
 func GetIPv6Networks(inventory *models.Inventory) (ret []string, err error) {
 	return getHostNetworks(inventory, func(i *models.Interface) []string { return i.IPV6Addresses })
 }
+
+func AreMachineNetworksIdentical(n1, n2 []*models.MachineNetwork) bool {
+	if len(n1) != len(n2) {
+		return false
+	}
+next:
+	for _, e1 := range n1 {
+		for _, e2 := range n2 {
+			if e1.Cidr == e2.Cidr {
+				continue next
+			}
+		}
+		return false
+	}
+	return true
```

Comment:
Could this just be:
```
return funk.Every(n1, n2...)
```
ref https://pkg.go.dev/github.com/thoas/go-funk#Every

Reply:
Or are we specifically dealing with a situation where the cluster id field could be different?

Reply:
Doesn't work if n2 is not []interface{} (Try it).
In addition, we ignore cluster id - it is not part of the payload and added later before save to the database.

Reply:
Well that's unfortunate. If anyone's interested I tried it out here https://go.dev/play/p/hsJmfNHRUO4.
I thought this would work with array parameter expansion given that it would work if we pass all the individual values in the array, but no such luck.